inst/paramtest/test_paramtest_classif.xgboost.R

```diff
@@ -42,7 +42,8 @@ test_that("classif.xgboost", {
     "label", # handled by mlr3
     "weight", # handled by mlr3
     "nthread", # handled by mlr3
-    "feval" # handled via eval_metric parameter
+    "feval", # handled via eval_metric parameter
+    "base_margin" # handled by mlr3
```

Comment:
why is `base_margin` added here?

Reply:
at least the comment "handled by mlr3" is confusing to me

Reply:
ha! there is some manual html magic to get the parameters but it's not added there so I manually add it to the list